integreat_cms/static/src/js/pois/poi-actions.ts

```diff
@@ -35,8 +36,43 @@ window.addEventListener("load", () => {
       latitude.dataset.initial = latitude.value;
     });
   }
+
+  document.getElementById("map_preview_button").addEventListener("click", showPreview);
 });
 
+function showPreview(){
+  const script = document.createElement("script");
+  script.src = "https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js";
```

Comment:
I don't think that's a good way to include the script - isn't it possible to import the things we need and let webpack bundle the stuff for us?